Source/Core/Core/HW/DSPHLE/DSPHLE.cpp

```diff
@@ -191,26 +191,37 @@ void DSPHLE::DSP_WriteMailBoxLow(bool cpu_mailbox, u16 value)
 // Other DSP functions
 u16 DSPHLE::DSP_WriteControlRegister(u16 value)
 {
-  DSP::UDSPControl temp(value);
```

Comment:
Okay, I'll keep the temp. Can you elaborate on what you mean by using a temp in DSP_ReadControlRegister? Do you mean that it should modify and return a temp, rather than modifying the member variable?